android/src/main/java/com/adyenreactnativesdk/util/ReactNativeError.kt

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2022 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ */
+package com.adyenreactnativesdk.util
+
+import android.util.Log
+import com.adyenreactnativesdk.component.BaseModuleException
+import com.adyenreactnativesdk.component.KnownException
+import com.adyenreactnativesdk.component.dropin.DropInException
+import com.facebook.react.bridge.ReadableMap
+import com.facebook.react.bridge.WritableMap
+import com.facebook.react.bridge.WritableNativeMap
+import java.lang.Exception
+
+object ReactNativeError {
+
+    private const val MESSAGE_KEY = "message"
+    private const val REASON_KEY = "reason"
+    private const val DISCRIPTION_KEY = "discription"
```

Comment:
Typo here

Reply:
Fixed in [b9edbc9](https://github.com/Adyen/adyen-react-native/pull/59/commits/b9edbc920769ed44b5d970ba3580c70b170480e9)